lib/workers/pr/body/updates-table.ts

```diff
@@ -1,29 +1,31 @@
 import { logger } from '../../../logger';
 import { regEx } from '../../../util/regex';
 import * as template from '../../../util/template';
-import type { BranchConfig } from '../../types';
+import type { BranchConfig, BranchUpgradeConfig } from '../../types';
 
 type TableDefinition = {
   header: string;
   value: string;
 };
 
-function getTableDefinition(config: BranchConfig): TableDefinition[] {
+function getRowDefinition(
+  prBodyColumns: string[],
+  upgrade: BranchUpgradeConfig
+): TableDefinition[] {
   const res: TableDefinition[] = [];
-  for (const header of config.prBodyColumns) {
-    const value = config.prBodyDefinitions[header];
+  for (const header of prBodyColumns) {
+    const value = upgrade.prBodyDefinitions[header];
```

Comment:
@rarkins can you please explain to me if that's a bug in another place? how come upgrade.prBodyDefinitions is undefined? 
where do i bring the definitions then? from the first row? like before? 
i'm not sure what to do here.

Reply:
You need to check the `generate.spec.ts` file tests, they probably missing some config objects.

Reply:
why `generate.spec.ts`? i ran the tests all passed successfully